test/base-index.js

```diff
@@ -107,18 +107,6 @@ test('encrypted', (t) => {
 
     db.get(privateMsg.key, (err, msg) => {
       t.equal(msg.content.text, 'super secret')
-      t.end()
-    })
-  })
-})
-
-test('db.close', (t) => {
```

Comment:
`db.close()` is no longer a separate API to be called, it's now automatically and internally done upon `sbot.close`. If you can think of a reason to close the db while keeping sbot up, let me know